ofrak_core/test_ofrak/unit/test_ofrak_server.py

```diff
@@ -1566,7 +1518,7 @@ async def test_open_project(ofrak_client: TestClient, test_project_dir):
 
 
 async def test_get_project_by_resource_id(ofrak_client: TestClient, test_project_dir):
-    git_url = "https://github.com/redballoonsecurity/ofrak-project-example.git"
+    git_url = "https://github.com/whyitfor/ofrak-project-example.git"
```

Comment:
Did you intend to change the example project URLs (here and above) to one of your own?

Reply:
This is needed b/c the fork updates the syntax to match this usage. Without this, the test fails.

Once the changes in this PR are good, we can try to merge in those changes.